### PR TITLE
Changed orgPolicy to COMPLEX from optional/mandatory

### PR DESCRIPTION
**Pull request by johnoneill46**
This hides the 'case assigned role' fields from having to be entered manually for the applicant or respondent organisations

Tested using https://manage-case.demo.platform.hmcts.net/cases/case-details/1603796357415746

Screenshot 2020-10-27 at 10 51 20

### CODE REVIEW

File: definitions/consented/json/CaseEventToFields/CaseEventToFields-respSol-nonprod.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "CaseEventID": "FR_amendApplicationDetails",
     "CaseFieldID": "RespondentOrganisationPolicy",
     "PageFieldDisplayOrder": 7,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "COMPLEX",
```
Comment (rayliangatos) [inline definitions/consented/json/CaseEventToFields/CaseEventToFields-respSol-nonprod.json:21]: needs to add this event FR_amendApplicationDetails to CaseEventToComplexType

File: definitions/consented/json/CaseEventToFields/CaseEventToFields-respSol-nonprod.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "CaseEventID": "FR_updateContactDetails",
     "CaseFieldID": "RespondentOrganisationPolicy",
     "PageFieldDisplayOrder": 27,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "COMPLEX",
```
Comment (rayliangatos) [inline definitions/consented/json/CaseEventToFields/CaseEventToFields-respSol-nonprod.json:34]: needs to add this event FR_updateContactDetails to CaseEventToComplexType, alike FR_solicitorCreate

File: definitions/contested/json/CaseEventToFields/CaseEventToFields-respSolJourney-nonprod.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "CaseEventID": "FR_amendApplication",
     "CaseFieldID": "RespondentOrganisationPolicy",
     "PageFieldDisplayOrder": 3,
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "COMPLEX",
```
Comment (rayliangatos) [inline definitions/contested/json/CaseEventToFields/CaseEventToFields-respSolJourney-nonprod.json:21]: needs to add this event FR_amendApplication to CaseEventToComplexType, alike FR_solicitorCreate

File: definitions/contested/json/CaseEventToFields/CaseEventToFields-shareACase-nonprod.json [hunk 4]
```diff
@@ -18,7 +18,7 @@
     "CaseEventID": "FR_newPaperCase",
     "CaseFieldID": "ApplicantOrganisationPolicy",
     "PageFieldDisplayOrder": 16,
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "COMPLEX",
```
Comment (rayliangatos) [inline definitions/contested/json/CaseEventToFields/CaseEventToFields-shareACase-nonprod.json:21]: needs to add this event FR_newPaperCase to CaseEventToComplexType, alike FR_solicitorCreate, if we change the type to COMPLEX

**Review by rayliangatos — APPROVED** [review]

**Review by shaunganley — APPROVED** [review]

**Review by ocarleton — APPROVED** [review]

**Pull request merged**